src/web/mjs/connectors/Nosajj.mjs

```diff
@@ -0,0 +1,52 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Nosajj extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'Nosajj';
+        super.label = 'Nosajj (shonanjunaigumi)';
```

Comment:
I'd rather use "shonanjunaigumi' as id and "Shonan Junai Gumi" as label, i think this website its better know by this name, but its personal taste

Reply:
id MUST be lowercase